src/scripts.js

```diff
@@ -12,8 +12,13 @@ import domUpdates from './domUpdates.js';
 const header = document.querySelector('#headingGreet');
 const tripCardContainer = document.querySelector('#gridContainer');
 const tripForm = document.querySelector('#formContainer');
+const tripFormDestinations = document.querySelector('#formDestinations');
+const estimateCost = document.querySelector('#getQuote');
+const formDuration = document.querySelector('#formDuration');
+const formTravelers = document.querySelector('#formTravelers');
+const pendingTripCost = document.querySelector('#pendingTripCost');
 
-let currentTraveler;
+let currentTraveler, destinationData;
```

Comment:
I had no idea we could use commas to declare several variables at once. I'll have to try this out.